backend/Dockerfile.cacheserver

```diff
@@ -1,5 +1,5 @@
 # Dockerfile for building the source code of cache_server
-FROM golang:1.11-alpine3.7 as builder
+FROM golang:1.13.15-alpine3.12 as builder
```

Comment:
Open question, I went with 1.13 since this is the version we use in https://github.com/kubeflow/pipelines/blob/ccc763fb718569a21b95a107ffe5beb023334742/go.mod#L95 but is this what we like to go with?

Reply:
Sounds reasonable to do smaller steps. If we need to upgrade go further, someone can raise a separate PR.